SettingPresets/Product_Platform/application_iOS.yml

```diff
@@ -1,2 +1,2 @@
-CODE_SIGN_IDENTITY: iPhone Developer
+CODE_SIGN_IDENTITY: Apple Development
```

Comment:
If this is the case should it be changed in the other presets as well?

Reply:
Does this change still support working for Xcode 10?
I think XcodeGen should switch presets by `xcodeVersion`.

Reply:
You're right about Xcode 10.  I'll take another pass at a more sophisticated implementation.